### ISSUE TITLE
401 unauthorized

### ISSUE THREAD

**Issue by cjauvin** [marksteve/flask-oauth2-login#1]
Hi, I'm getting:

```
{
  "error": "401 Client Error: Unauthorized"
}
```

when trying `example.py`, any idea? I have `http://localhost:5000/login/google` in my list of "Redirect URIs" on console.developers.google.com.

**Comment by irmiller22** [comment]
It's because the scopes that are provided in this are deprecated (see here: https://github.com/marksteve/flask-oauth2-login/blob/master/flask_oauth2_login/google.py#L10-13).

**Comment by marksteve** [comment]
Yes you're right. Sorry, I haven't been able to update this since I haven't been using it much. PRs are very welcome though :)

**Comment by SmartManoj** [comment]
Archive in favor of https://github.com/singingwolfboy/flask-dance
